### ISSUE TITLE
Sensors are disabled & nudes can't be found

### ISSUE THREAD

**Issue by sanderhoeppermans** [uvjim/linksys_velop#436]
Hi,

I installed the integration in home assistant via HACS. During the setup there is only one nude found, I filled in the IP-address of the primary nude. The other two nudes can't be found. 

After the installation was completed probably all the sensors are disabled (see screenshot). 

Does anyone now the solution for this? 

<img width="1183" alt="Scherm­afbeelding 2023-08-12 om 15 14 03" src="https://github.com/uvjim/linksys_velop/assets/121697071/55ff36a3-47c6-4a0f-950a-c6947c8cbcf0">

**Comment by uvjim** [comment]
Hi, it doesn't look like all sensors are disabled. The screenshot shows some that are enabled. As per the README, a number of entities are disabled by default. This is done because they aren't likely to be used by many people.  Just enable them if you want to use them.

**Comment by sanderhoeppermans** [comment]
> Hi, it doesn't look like all sensors are disabled. The screenshot shows some that are enabled. As per the README, a number of entities are disabled by default. This is done because they aren't likely to be used by many people. Just enable them if you want to use them.

Thanks for the reply! Maybe the problem is not the disabled integrations, but the integration itself. In the dashboard it shows that there are 0 online devices, and the other entities are unknown. 

<img width="308" alt="Scherm­afbeelding 2023-08-12 om 17 03 35" src="https://github.com/uvjim/linksys_velop/assets/121697071/011cc9ce-6d59-4d13-b164-58febf90c6c6">

There is also one nude found. I have one primaire nude and two secondary nodes

<img width="378" alt="Scherm­afbeelding 2023-08-12 om 17 02 55" src="https://github.com/uvjim/linksys_velop/assets/121697071/80adff39-3514-4d8f-b034-d66c50f6457f">

Do you know how I get the information from Linksys Velop in home assistant?

**Comment by uvjim** [comment]
There's not a huge amount I can look at without logs.  I suggest you delete the Linksys hub in Home Assistant and try again.  It should be automatically discovered (assuming that SSDP is allowed) and you should be using the password that you would use for local login to the router.

**Comment by uvjim** [comment]
Are you sure that you're using the local password for the router?  Not the smart wifi account password.

**Comment by sanderhoeppermans** [comment]
> Are you sure that you're using the local password for the router? Not the smart wifi account password.
> 
> **Update**: I will fix the error that you highlighted there in the upstream library but I'm not sure that it's the cause of what you are seeing

Yes, I use the same login as for the app and web page (https://linksyssmartwifi.com/ui/1.0.1.2197/dynamic/login.html). I tried other accounts, also without any result. 

It can't be the IP right? I use 192.168.1.1. This is also the IP of the primaire nude.

**Comment by sanderhoeppermans** [comment]
> Are you sure that you're using the local password for the router? Not the smart wifi account password.
> 
> **Update**: I will fix the error that you highlighted there in the upstream library but I'm not sure that it's the cause of what you are seeing

Yes, I tried every single password. All without result. When I login to: https://linksyssmartwifi.com/ui/1.0.1.2197/dynamic/login.html, it shows all the devices and nudes. 

I use exactly the same password as in the Linksys app (the local router passsword). Tried to change it, unfortunately without results. 

<img width="660" alt="Scherm­afbeelding 2023-08-16 om 16 43 41" src="https://github.com/uvjim/linksys_velop/assets/121697071/1ebfb0c0-5906-491b-84fe-86f10292a4a6">

**Comment by uvjim** [comment]
Which option do you choose here in the app? Email or router password? 

![Screenshot_20230816_155621_Linksys.png](https://github.com/uvjim/linksys_velop/assets/8431065/4c2bf8db-ba32-4b0f-802f-df23e7e7fdde)
